course_catalog/etl/loaders.py

```diff
@@ -81,12 +81,12 @@ def load_prices(resource, prices_data):
 
 
 def load_instructors(resource, instructors_data):
-    """Load the prices for a resource into the database"""
+    """Load the instructors for a resource into the database"""
     instructors = []
 
     for instructor_data in instructors_data:
         if "full_name" not in instructor_data:
-            instructor_data["full_name"] = None
+            instructor_data["full_name"] = instructor_data.get("title", None)
```

Comment:
Sorry, this turned out to be the wrong place.  It should go [here](https://github.com/mitodl/open-discussions/blob/master/course_catalog/serializers.py#L287-L306) instead (I'm not sure why there are so many places to assign instructors), something like this:

```python
            instructor = {
                "first_name": person.get("given_name", person.get("first_name")),
                "last_name": person.get("family_name", person.get("last_name")),
                "full_name": person.get("title"),
            }

            if person.get("salutation"):
                if instructor.get("full_name"):
                    instructor[
                        "full_name"
                    ] = "{salutation} {full_name}".format(
                        salutation=person["salutation"],
                        full_name=person["full_name"]
                    )
               elif instructor.get("last_name"):
                    instructor[
                        "full_name"
                    ] = "{salutation} {full_name}".format(
                        salutation=person.get("salutation"),
                        full_name=" ".join([part for part in [person.get("first_name"), person.get("middle_initial"), person.get("last_name")] if part])
                    )
```                    

A unit test should also be added to make sure it is working as expected.

Reply:
Thank you for pointing this out @mbertrand. 
I have updated the PR: Updated serializer (altered this code snippet you provided and separated it into util so it can be reused, if required), added a test to verify that it works as expected.
Apart from this, I haven't reverted the change made in `load_instructors` on the assumption that it should also store `title` as `full_name`. Is it correct or we should revert this change?

Reply:
I think it is probably fine to leave it here, I'll do some testing of it to make sure.